### PR TITLE
(closes #302) Fix ISO attachment list

### LINKED ISSUE #302: Iso attachment list is always empty

### PR DESCRIPTION
**Pull request by qqcky**
fix iso attachment list and reformat code of filtered template lists

### CODE REVIEW

File: src/app/template/template-filter-list/template-filter-list-selector.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  [searchPanelWhite]="selectedGroupings?.length"
+  [availableGroupings]="groupings"
+></cs-template-filters>
+
+<cs-template-list
+  *loading="fetching"
+  class="template-list"
+  [selectedTemplate]="selectedTemplate"
+  (selectedTemplateChange)="selectTemplate($event)"
+  (deleteTemplate)="removeTemplate($event)"
```
Comment (zolotyx) [inline src/app/template/template-filter-list/template-filter-list-selector.component.html]: Is it required? Where do we use it?

File: src/app/template/template-filter-list/template-filter-list-selector.component.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+export class TemplateFilterListSelectorComponent implements OnChanges {
+  @Input() public templates: Array<Template>;
+  @Input() public isos: Array<Iso>;
+
+  @Input() public dialogMode = true;
+  @Input() public selectedTemplate: BaseTemplateModel;
+  @Input() public showIsoSwitch = true;
+  @Input() public viewMode: string;
+  @Input() public zoneId: string;
+  @Output() public deleteTemplate = new EventEmitter();
```
Comment (zolotyx) [inline src/app/template/template-filter-list/template-filter-list-selector.component.ts]: Remove this also

File: src/app/template/template-filter-list/template-filter-list-selector.component.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  @Output() public selectedTemplateChange = new EventEmitter();
+  @Output() public viewModeChange = new EventEmitter();
```
Comment (zolotyx) [inline src/app/template/template-filter-list/template-filter-list-selector.component.ts]: Is it still required?

Comment (qqcky) [inline src/app/template/template-filter-list/template-filter-list-selector.component.ts]: I think, we use it in the vm creation, selecting the installation source

**Pull request merged**